### ISSUE TITLE
return all studies curated by a user

### ISSUE THREAD

**Issue by kcranston** [OpenTreeOfLife/otindex#33]
Need this to complete profile pages.

**Comment by kcranston** [comment]
doh - already implemented with find_studies and ot:curatorName property

**Comment by jimallman** [comment]
Is this fix running anywhere? I'm testing `find_studies` in the v3 API, and it still returns at most a single string value for `ot:tag` and `ot:curatorName`. (Apologies if we're still in transition from oti to otindex -- if so, let me know if I can help move that along.)

**Comment by mtholder** [comment]
My impression was that we were still using oti on tree+api.

**Comment by jimallman** [comment]
Drat. I was hoping to clean up the curator profile page with accurate study counts (added/curated/in-synthesis).

**Comment by jimallman** [comment]
Incidentally, it looks like [this force-to-unique-list trick](https://github.com/OpenTreeOfLife/otindex/blob/0080f50917e5d89a50c4874dfbbbe65e5381fbc6/otindex/scripts/load_nexson.py#L181) can have the unwanted(?) side effect of re-ordering the list of curator names. We depend on the first listed name to identify **the user who first added the study**. Is this information stored elsewhere, short of digging up GitHub versions and backtracking to the first entry?

**Comment by mtholder** [comment]
good catch that would definitely not be order preserving. I think the nexSON has a ot:curatorName field that is the unaltered list. So it may be OK that the index does not know the order.

I think:

    cur_set, cur_list = set(), []
    for c in curators:
        if c not in cur_set:
           cur_set.add(c)
           cur_list.append(c)
    curators = cur_list

is the efficient way to be order preserving.

**Comment by jimallman** [comment]
Yes, not a major problem, but I was hoping to use distinction in the curator-profile page, so show "added" vs. "curated" studies. (As an aside, feedback is welcome on how we describe these roles, and whether it's worth pointing out the distinction.) You can see this in the existing information about a curator's tree collections:

<img width="270" alt="screen shot 2018-04-17 at 5 23 24 pm" src="https://user-images.githubusercontent.com/446375/38898881-47ed35b4-4264-11e8-97e6-546948f09071.png">

**Comment by jimallman** [comment]
And to clarify, I get this information by walking the response to `find_studies`
